stories/components/FileCollection/FileCollection.md

```diff
@@ -0,0 +1,109 @@
+# FileCollection
+
+**Available components**:
+
+- [FileCollection](#filecollection)
+
+**How it is built**
+
+- uses react-dropzone library
+
+## KEY CONSIDERATIONS WHEN USING COMPONENT
+
+### UploadedFiles prop
+
+UploadedFiles prop is an array that can be passed as a json object. A key of 'keep' is mandatory to be passed in as this is what keeps track of whether or not the file should be kept or removed/deleted. If an uploaded file is removed/deleted through ui, `uploadedFile.keep = false`. If an uploaded file is not removed, `uploadedFile.keep = true`
+
+Example:
+
+```javascript
+  {
+    name: 'file1.txt',
+    src: 'https://www.google.ca',
+    file_type: 'text/plain',
+    keep: true,
+  },
+```
+
+### Handling Errors
+
+Error is not handled in the component. This comes from the fact that we are tracking whether or not an uploaded file should be kept or deleted.
+Error can be handled by checking if `newFile[].length > 0 || !values.uploadedFiles.every(file => file.keep === false)`
+
+Example
+
+```javascript
+const fileCollection = () => {
+  const fileCollectionValid = values =>
+    values.files.length > 0 || !values.uploadedFiles.every(file => file.keep === false);
+
+  const handleSubmit= (formValues) => {
+    if (!fileCollectionValid(values)) {
+      return;
+    }
+    console.log(values);
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <FileCollection
+        newFiles={files} // initial empty []
+        data={uploadedFiles}
+        ...
+      />
+      {!fileCollectionValid(values) && (
+        <div style={{ color: 'red' }}>Please uploaded atleast one file</div>
+      )}
+    </form>
+  )
+}
+
+```
+
+## FileCollection
+
+Files can be dragged and dropped or browsed and muli-selected. Files array passed as prop pre-populate in a list.
+
+### Props
+
+| name            | type   | default | description                                                                                                                                         |
+| --------------- | ------ | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
nit: Believe the amount of -'s under the description doesn't need to be that long

Reply:
I think it makes it look nicer in the code editor if you toggle word wrap